docs/en/guides/pipeline-integrations/development/sgtk-developer-app.md

```diff
@@ -0,0 +1,257 @@
+---
+layout: default
+title: Developing apps
+pagename: sgtk-developer-app
+lang: en
+---
+
+# Developing your own app
+
+## Introduction
+
+This guide outlines what a Toolkit app is, and covers how to create your own, and also explains some of the basics around app development.
```

Comment:
Why not clean this up? 

"This guide outlines what a Toolkit app is, covers how to create one, and explains some basics around app development."